### ISSUE TITLE
[BUG REPORT] Dropping unmatched japanese characters even though font clearly supports japanese characters. 

### ISSUE THREAD

**Issue by Dragonsight91** [polybar/polybar#2493]
## Checklist

* [x] I have read the appropriate section in the [contributing
  guidelines](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md)
* [x] I believe this issue is a problem with polybar itself and not a misconfiguration on my part.
* [x] I have searched for other open and closed [issues](https://github.com/polybar/polybar/issues?q=is%3Aissue) that
  may have already reported this problem.
* [x] I have checked the [known issues](https://github.com/polybar/polybar/wiki/Known-Issues) page for this problem.

## Describe the bug
quite simple: When using ANY nerd font, Japanese characters seem to get dropped.
i tested `Droid font mono`, `Daddy time mono` and `SauceCodePro` 

**Expected behavior:**
Characters are all supported by the fonts i use, so they should just be displayed.

**Actual behavior:**
Characters do not show 

**Was it working before?**
* Did you also experience this bug in an earlier version of polybar ? 
    Unknown, i just noticed

## To Reproduce

uses: i3 module, but can just insert japanese characters into any label

```dosini
[bar/root]
width = 100%
height = 27
fixed-center = true
bottom=true

line-size = 3
line-color = #f00

border-size = 0
border-color = #00000000

padding-left = 0
padding-right = 2

module-margin-left = 1
module-margin-right = 2

font-0 = "DaddyTimeMono Nerd Font:style=Book:pixelsize=12;"
; some of the dropped text to show the same font works elsewhere
; 我がまま
; i also don't use ANY other fonts, only font-0 and only that font

[bar/bar-center]
inherit = bar/root
modules-left = i3

[module/i3]
type = internal/i3
format = <label-state> <label-mode>
label-focused = まtest%name%  ; adding a character that gets dropped and ones that don't
label-unfocused = ${self.label-focused}
label-visible = ${self.label-focused}
label-urgent = ${self.label-focused}
```

Font: [DaddyTimeMono Nerd Font](https://github.com/ryanoasis/nerd-fonts/releases/download/v2.1.0/DaddyTimeMono.zip)

## Polybar Log

```
polybar|notice:  Parsing config file: /home/emma/.config/polybar/config
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/config
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/filesystem.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/i3.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/memory.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/wifi.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/ethernet.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/date.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/spotify.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/battery.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/modem.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/pulseaudio.module
polybar|trace: config_parser: Parsing /home/emma/.config/polybar/modules/bluetooth.module
polybar|trace: config: Inheriting keys from "bar/root" in "bar/bar-center"
polybar|trace: config: Inheriting keys from "bar/root" in "bar/bar-left"
polybar|trace: config: Inheriting keys from "bar/root" in "bar/bar-right"
polybar|trace: config: Inheriting keys from "module/battery-root" in "module/battery-one"
polybar|trace: config: Inheriting keys from "module/battery-root" in "module/battery-two"
polybar|info:  Environment var reference ${SCREEN_CENTER} found (value=DisplayPort-1)
polybar|info:  Loaded monitor DisplayPort-1 (1920x1080+1920+0)
polybar|info:  Bar geometry: 1920x27+1920+1053; Borders: 0,0,0,0
polybar|trace: bar: Attach X event sink
polybar|trace: bar: Attach signal receiver
polybar|trace: controller: Install signal handler
polybar|trace: controller: Setup user-defined modules
polybar|trace: module/wlan: Deconstructing
polybar|error: Disabling module "wlan" (reason: Invalid network interface "wlp4s0")
polybar|trace: module/battery-one: Deconstructing
polybar|error: Disabling module "battery-one" (reason: No suitable way to get current charge state)
polybar|trace: module/battery-two: Deconstructing
polybar|error: Disabling module "battery-two" (reason: No suitable way to get current charge state)
polybar|trace: pulseaudio: started mainloop
polybar|notice:  pulseaudio: using default sink bluez_sink.08_EF_3B_96_F4_28.a2dp_sink
polybar|info:  Starting application
polybar|trace: controller: Main thread id = 1
polybar|info:  Starting module/i3
polybar|info:  Starting module/spotify
polybar|info:  Starting module/memory
polybar|trace: module/i3: Thread id = 2
polybar|info:  module/spotify: Invoking shell command: "~/.config/polybar/scripts/spotify_scroll.sh"
polybar|info:  Starting module/filesystem
polybar|trace: module/memory: Thread id = 3
polybar|info:  Starting module/eth
polybar|trace: module/filesystem: Thread id = 4
polybar|info:  Starting module/bluetooth
polybar|trace: module/eth: Thread id = 5
polybar|info:  Starting module/pulseaudio
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 861860 to finish...
polybar|trace: module/pulseaudio: Thread id = 6
polybar|info:  Starting module/date
polybar|info:  Entering event loop (thread-id=1)
polybar|trace: controller: Attach config watch
polybar|trace: module/date: Thread id = 7
polybar|info:  Eventqueue worker (thread-id=8)
polybar|trace: bar: Create renderer
polybar|trace: renderer: Get TrueColor visual
polybar|trace: renderer: Allocate colormap
polybar|trace: renderer: Allocate output window
polybar|trace: renderer: Allocate window pixmaps
polybar|trace: renderer: Allocate graphic contexts
polybar|trace: renderer: Allocate alignment blocks
polybar|trace: renderer: Allocate cairo components
polybar|trace: renderer: Load fonts
polybar|info:  Configured DPI = 96x96
polybar|notice:  Loaded font "DaddyTimeMono Nerd Font:style=Book:pixelsize=12" (name=DaddyTimeMono Nerd Font, offset=0, file=/home/emma/.fonts/DaddyTimeMono Nerd Font Complete.ttf)
polybar|info:  Bar window: 0x0800002
polybar|trace: bar: Reconfigure window
polybar|trace: bar: Set window WM_NAME
polybar|trace: bar: Set window _NET_WM_WINDOW_TYPE
polybar|trace: bar: Set window _NET_WM_STATE
polybar|trace: bar: Set window _NET_WM_DESKTOP
polybar|trace: bar: Set window _NET_WM_PID
polybar|trace: bar: Map window
polybar|trace: bar: Draw empty bar
polybar|trace: bar: Setup tray manager
polybar|info:  Disabling tray manager (reason: missing `tray-position`)
polybar|info:  module/i3: Rebuilding cache
polybar|info:  module/spotify: Rebuilding cache
polybar|info:  module/memory: Rebuilding cache
polybar|info:  module/filesystem: Rebuilding cache
polybar|info:  module/eth: Rebuilding cache
polybar|info:  module/bluetooth: Rebuilding cache
polybar|info:  module/pulseaudio: Rebuilding cache
polybar|info:  module/date: Rebuilding cache
polybar|trace: bar: Force update
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|trace: bar: Received expose event
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(3 geom=684x27+1236+0, falloff=0)
polybar|trace: command: Exited with status 0
polybar|info:  module/bluetooth: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/spotify: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 861953 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  module/eth: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862049 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862139 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862222 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/eth: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862305 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862388 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/i3: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/i3: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862478 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  module/eth: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862562 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862645 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862728 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/eth: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862813 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862901 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 862988 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  module/eth: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=666x27+1254+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863071 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863155 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=666x27+1254+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863252 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/eth: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863347 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863430 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863520 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  module/eth: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863603 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863690 to finish...
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|info:  Redrawing bar window
polybar|warn:  Dropping unmatched character ま (U+307e) in '  まtestEditor  '
polybar|warn:  Dropping unmatched character 誤 (U+8aa4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 解 (U+89e3) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 加 (U+52a0) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 藤 (U+85e4) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 有 (U+6709) in '誤解 - 加藤有紀'
polybar|warn:  Dropping unmatched character 紀 (U+7d00) in '誤解 - 加藤有紀'
polybar|trace: renderer: flush(1 geom=126x27+0+0, falloff=0)
polybar|trace: renderer: flush(2 geom=63x27+929+0, falloff=0)
polybar|trace: renderer: flush(3 geom=828x27+1092+0, falloff=0)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863773 to finish...
polybar|info:  module/eth: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|trace: command: Exited with status 0
polybar|info:  module/memory: Rebuilding cache
polybar|trace: bar: Ignoring update (unchanged)
polybar|info:  module/bluetooth: Invoking shell command: "~/.config/polybar/scripts/rofi-bluetooth.sh --status"
polybar|trace: command: Waiting for pid 863856 to finish...
polybar|trace: command: Exited with status 0
```

## Screenshots

![an image showing gucharmap with the font in question and an example of the characters not showing up](https://user-images.githubusercontent.com/42149402/132498680-e06f9823-9359-4d8a-87f5-e81d358c9d81.png)

## Environment:
* WM: i3
* Distro: Arch Linux
* Output of `polybar -vvv`:
```
polybar 3.5.6

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: Release
Compiler: /bin/c++
Compiler flags: -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection -Wp,-D_GLIBCXX_ASSERTIONS -Wall -Wextra -Wpedantic -O3 -DNDEBUG
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now 
```

**Comment by patrick96** [comment]
> Characters are all supported by the fonts i use

This assumption is wrong. As far as I know, nerd fonts don't include any japanese characters. I downloaded the fonts you linked and indeed, they do not contain glyphs for japanese characters. 

In the `gucharmap` screenshot you showed, did you select `View -> Show only glyphs from this font`?

`M+` is the only nerd font I found that has japanese characters and that's because the base font itself is for japanese characters.
I would recommend using `Noto Sans CJK` for japanese characters (also for chinese and korean)
